Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -35,4 +53,10 @@ void BatteryStateOfChargeService::addData(const BatteryData& batteryData)
     // This is where you can update your variables
     // Hint: There are many different ways that the totalAmpHoursUsed can be updated
     // i.e: Taking a running average of your data values, using most recent data points, etc.
+
+    double averageCurrent = (dataPoint_.current + batteryData.current) / 2;
```

Comment:
What happens here on the very first run? IE dataPoint is some garbage value

Reply:
I think its just equal to 0?